### ISSUE TITLE
Crashing When Using "Display Options -> Percent" 

### ISSUE THREAD

**Issue by caleb-petry** [PardeeCenterDU/IFs-Issues-Tracking#232]
Crash When Using "Display Options -> Percent" 

Date: 2023/01/11
Version: 7.92 .Net Development - Rebased to 1995
Branch: Threat
Found by: Caleb Petry
Error Type: Blowup

Model crashes when any other option is selected after turning on "Percent" in "Display Options" in the Flex Display. Does not seem to occur with all variables at all times; "Capital Stock - Billion dollars" seems to occur the most consistently. 

![image](https://user-images.githubusercontent.com/7695597/211907497-772b66f9-83fe-414d-8c39-51944e288e39.png)
 `The following error occurred:Exception of type 'System.Web.HttpUnhandledException' was thrown.System.InvalidCastException: Conversion from string "" to type 'Double' is not valid. ---> System.FormatException: Input string was not in a correct format.at Microsoft.VisualBasic.CompilerServices.Conversions.ParseDouble(String Value, NumberFormatInfo NumberFormat)at Microsoft.VisualBasic.CompilerServices.Conversions.ToDouble(String Value, NumberFormatInfo NumberFormat) --- End of inner exception stack trace ---at Microsoft.VisualBasic.CompilerServices.Conversions.ToDouble(String Value, NumberFormatInfo NumberFormat)at Microsoft.VisualBasic.CompilerServices.Operators.CompareObject2(Object Left, Object Right, Boolean TextCompare)at Microsoft.VisualBasic.CompilerServices.Operators.ConditionalCompareObjectGreaterEqual(Object Left, Object Right, Boolean TextCompare)at IFsWeb.frm_TableDisplay.Convert_Billions_To_Trillions() in C:\Users\Public\IFs\IFsWeb\frm_TableDisplay.aspx.vb:line 10883at IFsWeb.frm_TableDisplay.frm_TableDisplay_Load(Object eventSender, EventArgs eventArgs) in C:\Users\Public\IFs\IFsWeb\frm_TableDisplay.aspx.vb:line 1896at System.Web.UI.Control.OnLoad(EventArgs e)at System.Web.UI.Control.LoadRecursive()at System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)`

**Comment by PardeeCenterIFs** [comment]
Using the 2017 base, I loaded this list with the default options, since you don't mention anything, so that's Afghanistan, Agriculture, go to Table, change to Percentage, and it works fine.

The error mentions the Convert_Billions_To_Trillions functions which indicates big values, so I tried it again with World, and Total, and work fine again.

Can you send an image that includes the table before the error.

**Comment by caleb-petry** [comment]
Of course, here is the table right before the error. From here, actions such as turning off the percent option cause the initial error message.

![image](https://user-images.githubusercontent.com/7695597/212387473-2dccfa7e-42e7-474e-937d-4a7aec522d9b.png)

**Comment by PardeeCenterIFs** [comment]
It looks like you're using History plus Forecast, right? Not plainly "Capital Stock - Billion dollars"

**Comment by caleb-petry** [comment]
That is due to another issue #233, though I could see how they might be related.

**Comment by caleb-petry** [comment]
This one still occurs sometimes with 7.93's GUI, but overall is more stable now.

It seems to occur more often when the model has been used for a while.

**Comment by caleb-petry** [comment]
Not a recurrent issue for my work anymore.
